Cargo.toml

```diff
@@ -71,17 +71,17 @@ zeroize = { opt-level = 3 }
 
 [patch.crates-io]
 # frontier deps
-fc-consensus = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-fc-rpc = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-fc-rpc-core = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-fp-consensus = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-fp-evm = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-fp-rpc = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-ethereum = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm-precompile-ed25519 = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm-precompile-modexp = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm-precompile-simple = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
+fc-consensus = { git = "https://github.com/VLT-Protocol/frontier.git", rev = "6a284edd6bf34e96d1a4721fad215a0adb2cd5d3" }
```

Comment:
This fork is potentially an issue. Have you tested this against a live chain? Connected to beresheet/mainnet using this new build?

Have you synced from scratch on both beresheet/mainnet?

Reply:
There are fixes in our fork that make it possible to process the initial runtime upgrade that may be missing in this fork. Without testing a full sync, I do not feel comfortable merging this in. Please create cloud instances and try to sync and circle back.

Reply:
@raykyri did you have a fork from VLT when you made the patch

Reply:
This is the only change in the VLT branch:
https://github.com/VLT-Protocol/frontier/commit/6a284edd6bf34e96d1a4721fad215a0adb2cd5d3